tests/push/test_email.py

```diff
@@ -67,7 +67,7 @@ def make_homeserver(self, reactor, clock):
         config["public_baseurl"] = "aaa"
         config["start_pushers"] = True
 
-        hs = self.setup_test_homeserver(config=config)
+        self.hs = self.setup_test_homeserver(config=config)
```

Comment:
`setUp` (the thing that calls `make_homeserver` sets `self.hs`, so this is unnecessary.

Reply:
Aha, indeed, good point!